.github/workflows/iroha2-dev-pr-label.yml

```diff
@@ -13,6 +13,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
```

Comment:
This is a fix for #2708 after observing an [error](https://github.com/hyperledger/iroha/runs/8228305240?check_suite_focus=true#step:3:428)

Reply:
Did you try it? It still seems as the same error.

Reply:
https://github.com/hyperledger/iroha/pull/2680#discussion_r964673683